repo/fsrepo/fsrepo.go

```diff
@@ -482,6 +482,14 @@ func (r *FSRepo) SetConfigKey(key string, value interface{}) error {
 		return err
 	}
 
+	// Load private key to guard against it being overwritten.
+	// NOTE: this is a temporary measure to secure this field until we move
+	// keys out of the config file.
+	pkval, err := common.MapGetKV(mapconf, "Identity.PrivKey")
```

Comment:
- What if the name of the fields in the structs changes and people forget to change it here? should be a const.